restaurant/src/assets/home.css

```diff
@@ -0,0 +1,45 @@
+.home-page {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  background-image: url("bg1.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  min-height: 100%;
+}
+.Homeheader {
```

Comment:
Please keep your naming lowercase and consistent. Words in CSS should be separated by `-` symbol, like `home-header`